src/event/flatten.rs

```diff
@@ -1,5 +1,6 @@
-use crate::event::{LogEvent, ValueKind};
+use crate::event::{Atom, LogEvent, ValueKind};
 use serde_json::{Map, Value};
+use std::convert::AsRef;
```

Comment:
I believe this is always included in the `std::prelude` so we don't need it here.

Reply:
True